chain/gen/gen.go

```diff
@@ -526,9 +526,11 @@ func IsRoundWinner(ctx context.Context, ts *types.TipSet, round int64, miner add
 		return nil, xerrors.Errorf("failed to look up miners sector size: %w", err)
 	}
 
+	snum := types.BigDiv(pow.MinerPower, types.NewInt(ssize))
```

Comment:
could also take the length of 'sectors'

Reply:
yeah. Given potential upcoming changes around spec (ie if the sector's entire size is not used for power), that may actually be a better solution. Should I rePR?